File: javascript/control_flow/if_statement/if_statement.js
#!/usr/bin/env node

if (1 > 2)
  console.log('1 > 2');

if (2 > 1)
  console.log('2 > 1');

console.log();

let condition = false;

if (condition) {
  console.log('false: ok');
}

condition = true;

if (condition) {
  console.log('true: ok');
}

console.log();

File: perl/control_flow/if_statement/if_statement.pl
#!/usr/bin/perl

use strict;
use warnings;

if (1 > 2) {
  print("1 > 2\n");
}

if (2 > 1) {
  print("2 > 1\n");
}

print("\n");

my $condition = 0;

if ($condition) {
  print("False (0): ok\n");
}

$condition = 1;

if ($condition) {
  print("True (1): ok\n");
}

print("\n");
